lib/adapters/websql/bulkDocs.js

```diff
@@ -18,7 +18,7 @@ var ATTACH_AND_SEQ_STORE = websqlConstants.ATTACH_AND_SEQ_STORE;
 var select = websqlUtils.select;
 var stringifyDoc = websqlUtils.stringifyDoc;
 var compactRevs = websqlUtils.compactRevs;
-var unknownError = websqlUtils.unknownError;
```

Comment:
`unknownError` is the wrong name for the this thing; it is a generic handler for WebSQL errors.